test/k8s/verifier.go

```diff
@@ -93,7 +93,7 @@ var _ = Describe("K8sDatapathVerifier", func() {
 	}
 
 	getKernel := func() string {
-		kernel := "49"
```

Comment:
We're still running the verifier job on 4.9 (as https://github.com/cilium/cilium/pull/22754/commits/393e1004c4af51346fc51f7bfb5250b4ba108d8b hasn't been backported yet).

But this change means that it now uses the 5.4 complexity configs. Note how eg https://jenkins.cilium.io/job/Cilium-PR-K8s-1.19-kernel-4.9/2628/testReport/junit/Suite-k8s-1/19/K8sDatapathVerifier_Runs_the_kernel_verifier_against_Cilium_s_BPF_datapath/ contains `-DENABLE_EGRESS_GATEWAY=1` - that doesn't exist in a 4.9 config.